src/util/report.py

```diff
@@ -319,9 +319,11 @@ def ref(xid):
                 _logger.warning("Cannot unfold chat window", exc_info=True)
 
 
-def get_anchor_link_to_record(model, id, name, action_id=None):
+def get_anchor_link_to_record(model, id, name=None, action_id=None):
```

Comment:
```suggestion
def get_anchor_link_to_record(model, id, name, action_id=None):
```

We handle cases when `name` turns out to be Falsey, but I don't think we want the argument to be optional.

cc: @aj-fuentes

Reply:
Ah, yes, indeed. I think that was the intent. But I think we can add the default to None... I don't see any downside. @KangOl wdyt?